Atomic/sign.py

```diff
@@ -0,0 +1,107 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+import re
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # Atomic is run as sudo. Should we work around that?
```

Comment:
I don’t understand what this comment is trying to say. Is `atomic(1)` _always_ run through sudo? (Probably not.) Does `atomic sign` always need to be run through sudo? (That would be surprising to me.)

Assuming “as sudo” means “as root”, this comment should say why is that needed so that it is clearer what to work around.